views/templates/admin/backend.tpl

```diff
@@ -75,13 +75,13 @@
                 <p><strong>{l s='1. You do not have an Alipay Business Account:' mod='alipay'}</strong></p>
                 <p class="justify padding-left">
                     {l s='Visit' mod='alipay'}<br />
-                    <a class="alipay-website" href="http://global.alipay.com/product/websitepayment.html" target="_blank">http://global.alipay.com/product/websitepayment.html</a><br />
+                    <a class="alipay-website" href="http://global.alipay.com/product/websitepayment.html?from=prestashop" target="_blank">http://global.alipay.com/product/websitepayment.html?from=prestashop</a><br />
```

Comment:
The only link that need to have tracking is on the href statement \<a href="here">\</a> also you should change the name of the link to have something more clean on your back office.

![download](https://cloud.githubusercontent.com/assets/6247743/11467335/a7f2d7fe-9747-11e5-81da-e89a63abb201.png)